### DISCUSSION TITLE
DJ Support without AutoDJ?

### DISCUSSION THREAD

**Discussion by Radiotom** [AzuraCast/AzuraCast#6729] [General]
Hey there, 
i have installed Azuracast for Radio Project.
I need a configuration for Live Djs (and Recordings) but without activate the Auto DJ! or LS ?

Thats important because the Stream has to be stoped after disconecting from the DJ.
Over all i would like to enable the Recording funktion of each DJ. 

how is it possible?

Thanks.

**Comment by BusterNeece** [comment]
@Radiotom We use the same software to accept incoming DJ connections that we use to run the AutoDJ (Liquidsoap). We don't currently support accepting DJ connections directly on the Icecast/Shoutcast end, as that would significantly impact how we handle metadata and authentication. You can always connect to Icecast using the `source` password we generate for you, but our built-in DJ tools (including the ability to record live broadcasts) depend on Liquidsoap to function.

**Comment by Radiotom** [comment]
Ok thats not good. 
I tried to manage multiple Stations (source connect Dj) with one another "Recording Station" include a fake AutoDJ that only play random titles. At next I will connect the source  stations as localhost to the recording station via remote Relay. 
But it seems not work. 

Before I spend a lot of hours if it work, can u say that's possible?
